nanome/api/files.py

```diff
@@ -67,7 +71,7 @@ def cb(error, file):
                     ofile.close()
             callback(error, dest)
         id = self.plugin._network._send(_Messages.get, source, True)
-        self.plugin._save_callback(id, cb)
+        return self.plugin._save_callback(id, cb)
```

Comment:
This one isn't quite async; there's a local callback to write the file to `dest`. Not sure how to get around this